pkg/plugin/manager.go

```diff
@@ -45,9 +45,9 @@ type Manager struct {
 }
 
 // NewManager instantiates a manager struct.
-func NewManager(storeMS *store.Store) (*Manager, error) {
-	//nolint:exhaust,exhaustruct
-	manager := &Manager{plugins: make(map[string]*Plugin), authorNameToID: make(map[string]string), store: storeMS}
+func NewManager() (*Manager, error) {
+	//nolint:exhaust,exhaustruct,lll
+	manager := &Manager{plugins: make(map[string]*Plugin), authorNameToID: make(map[string]string), store: store.StoreInstance}
```

Comment:
i don't think you need to give the store instance to the manager here. you can just call it from everywhere its needed